config/express.js

```diff
@@ -47,7 +47,7 @@ if (config.env === 'development') {
 }
 
 // mount all routes on /api path
-app.use('/api', routes);
+app.use('/', routes);
```

Comment:
Yes. I have definitely been a part of the debate on more than one occasion. Again, my take, FWIW, is that we can use the LB to define any of these rules. If we end up with different versions of the API service, we can run them simultaneously in different containers and map requests to one or the other based on either a header or a URL component, as we choose. We shouldn't have to include the rule in our Express routes.